### PR TITLE
feat(Manager) : Fixing API Key onsubmit bug | renaming documents tab and select my property

### PR DESCRIPTION
**Pull request by nikhitamore**
## Explain the feature/fix

Following bug fixes 

1. Fixing create API key form on submit with error issue
2. Renaming documents
3. changing select web property to my property and handling the redirection
4. shifting pagination for list of spas to top

## Screenshot(s)

<details>
<summary>View Screenshots</summary>

</details>

### Ready-for-merge Checklist

- [ ] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Does the change have appropriate unit tests?
- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [ ] Was this feature demo'd and the design review approved?

### CODE REVIEW

File: packages/manager/src/layouts/AppLayout/components/SideBar/SideBar.tsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ type SNProps = {
   isActive: boolean;
 };
 
-function getIdentifier(identifier: string) {
+function GenereateIdentifier(identifier: string) {
```
Comment (soumyadip007) [inline packages/manager/src/layouts/AppLayout/components/SideBar/SideBar.tsx:28]: can we make it as `genereateIdentifier`

File: packages/manager/src/layouts/AppLayout/components/SideBar/SideBar.tsx [hunk 2]
```diff
@@ -111,13 +111,13 @@ export const SideBar = () => {
...
               <NavExpandable
-                title={selectedProperty || 'Select Web Property'} // Use the selectedProperty state
+                title={selectedProperty || 'Select My Property'} // Use the selectedProperty state
                 groupId="nav-expanded-web-property"
                 isExpanded={isExpanded}
               >
                 {filteredWebProperties?.map((property) => (
                   <NavItem key={property.title}>
-                    <Link href={`/properties/${getIdentifier(property.title)}`} passHref>
+                    <Link href={`/properties/${GenereateIdentifier(property.title)}`} passHref>
```
Comment (soumyadip007) [inline packages/manager/src/layouts/AppLayout/components/SideBar/SideBar.tsx:120]: make it as `genereateIdentifier`

**Review by soumyadip007 — CHANGES_REQUESTED** [review]
@nikhitamore please make these changes

File: packages/manager/src/layouts/AppLayout/components/SideBar/SideBar.tsx (hunk 1 above)
Comment (nikhitamore) [reply packages/manager/src/layouts/AppLayout/components/SideBar/SideBar.tsx]: done

File: packages/manager/src/layouts/AppLayout/components/SideBar/SideBar.tsx (hunk 2 above)
Comment (nikhitamore) [reply packages/manager/src/layouts/AppLayout/components/SideBar/SideBar.tsx]: done

**Review by soumyadip007 — APPROVED** [review]
LGTM

**Pull request merged**